src/generic_chunk.h

```diff
@@ -25,11 +26,21 @@ typedef enum {
     CHUNK_COMPRESSED 
 } CHUNK_TYPES_T;
 
+typedef struct AddCtx {
```

Comment:
The name is confusing, should be renamed to reflect that its being used only in upserts.